src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php

```diff
@@ -126,6 +131,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->validateInput($input);
         $object = $this->getContainerBuilder();
 
+        if ($input->getOption('deprecations')) {
```

Comment:
what about the `format` option at this point? Should we throw for non-text? Would it be better to handle the logic in the current descriptor(s)?

Reply:
IMHO we could say in the doc that it's only in text format atm. Adding this to the descriptors will not be the same in term of complexity, I don't know if we really want to see this in all format.

cc @nicolas-grekas

Reply:
we already have the descriptor infrastruxture + we only have to implement txt IMHO

Reply:
agree with @ro0NL, you should use the current descriptor and implement the supported formats.

Reply:
updated ;)